src/macaron/config/defaults.ini

```diff
@@ -211,6 +211,52 @@ jenkins =
     gradle-git-publish
     gitPublishPush
 
+# This is the spec for trusted Pip packaging tools.
+[builder.pip]
+entry_conf =
+    setup.py
+    setup.cfg
+    pyproject.toml
+build_configs =
+packager =
+    pip
+    pip3
+    flit
+    conda
+publisher =
+    twine
+    flit
+    conda
+builder_module =
+    python
+    python3
+build_arg =
+    install
+    build
+    setup.py
+deploy_arg =
+    publish
+    upload
+[builder.pip.ci.deploy]
+github_actions = pypa/gh-action-pypi-publish
+
+# This is the spec for trusted Poetry packaging tools.
+[builder.poetry]
+entry_conf = pyproject.toml
+build_configs = poetry.lock
+builder =
+    poetry
+    poetry-core
+builder_module =
```

Comment:
```suggestion
interpreter = 
```

Reply:
Changed in [9c167f2](https://github.com/oracle-samples/macaron/pull/67/commits/9c167f271c3b880c3db95d6de5fc7bb51bdfcf3d).